### ISSUE TITLE
Make a new release

### ISSUE THREAD

**Issue by vitlav** [4lex4/scantailor-advanced#146]
Please make new release (set new tag and release tarball).

**Comment by thom319** [comment]
I further this.

**Comment by kms54321** [comment]
Yes, please! This is a unique and awesome tool that keeps getting better.

**Comment by dicktyr** [comment]
indeed, some distros package only tagged releases
and this software is very useful so please tag a new release

**Comment by rudolphos** [comment]
Bump.
This is the best version of ScanTailor. There is also newer version ScanTailor Universal, but it lacks many features from output section. Though it has jp2 support through libopenjp2.

**Comment by rudolphos** [comment]
Bump.
This is the best version of ScanTailor. There is also newer version ScanTailor Universal, but it lacks many features from output section. But it has an experimental jp2 support through libopenjp2.
